docs/install.rst

```diff
@@ -14,25 +11,37 @@ Before using Tidy3D, you must first `sign up <https://tidy3d.simulation.cloud/si
 
 By signing up for a free account, you can obtain an API key `here <https://tidy3d.simulation.cloud/account?tab=apikey>`_. You can also `manage your simulation jobs <https://tidy3d.simulation.cloud/folders>`_ and access `graphic user interface <https://tidy3d.simulation.cloud/workbench?taskId=pa-94c49911-132d-48bc-8ec0-f0a4e55140a3>`_ if needed.
 
-Installation of Tidy3D Python API
----------------------------------
+Managing API Keys
+------------------
 
-If you wish to install the Tidy3D Python API locally, the following instructions should work for most users.
+Quick Configuration
+~~~~~~~~~~~~~~~~~~~
 
-.. code-block:: bash
+.. tabs::
 
-    pip install --user tidy3d
-    tidy3d configure --apikey=XXX
+    .. group-tab:: Local Installation |:computer:|
 
-Where ``XXX`` is your API key, which can be copied from your `account page <https://tidy3d.simulation.cloud/account>`_ in the web interface.
+        If you wish to install the Tidy3D Python API locally, the following instructions should work for most users.
 
-In a hosted jupyter notebook environment (eg google colab), it may be more convenient to install and configure via the following lines at the top of the notebook.
+        .. code-block:: bash
 
-.. code-block:: bash
+            pip install --user tidy3d
+            tidy3d configure --apikey=XXX
+
+    .. group-tab:: Any Hosted Environment |:cloud:|
+
+        Where ``XXX`` is your API key, which can be copied from your `account page <https://tidy3d.simulation.cloud/account>`_ in the web interface.
+
+        In a hosted jupyter notebook environment (eg google colab), it may be more convenient to install and configure via the following lines at the top of the notebook.
+
+        .. code-block:: bash
```

Comment:
should be python syntax highlighting

Reply:
My bad fixed them all

Reply:
a lot of them were actually supposed to be bash, only noticed this one.